packages/expo-clipboard/src/Clipboard.ts

```diff
@@ -43,6 +43,94 @@ export function setString(text: string): void {
   return ExpoClipboard.setString(text);
 }
 
+/**
+ * (iOS only) Gets the url from the user's clipboard.
```

Comment:
Instead of `(iOS only)` in the description, please use `@platform ios` annotation. 

To update the docs, you will also need to run the command below and commit changes to the JSON data file:
```
et gdad -p expo-clipboard
```

You can check the new method docs correctness by running website locally and selecting "Unversioned" as an API version.